descriptors/ModuleDescriptor-template.json

```diff
@@ -14,7 +14,7 @@
   "provides": [
     {
       "id": "notes",
-      "version": "2.0",
+      "version": "3.0",
```

Comment:
As the APIs provided by this module are now incompatible with the APIs provided by previous versions, this is a major implementation breaking change.

That should be reflected in a major version change for the module.